lib/Db/Message.php

```diff
@@ -232,11 +250,14 @@ public function setFlag(string $flag, bool $value = true) {
 			// Ignore
 			return;
 		}
-
-		$this->setter(
-			$this->columnToProperty("flag_$flag"),
-			[$value]
-		);
+		if ($flag === '$label1') {
```

Comment:
how about a `\OCA\Mail\Db\Tag::LABEL_IMPORTANT` for better readability and lower chances of typos?